Documentation/DataSources/DataSources.md

```diff
@@ -63,6 +64,16 @@ To make a query control use this data source, use the _Query Type_ drop down to
 
 ![A image of a workbook with Azure resource health data](../Images/ResourceHealthDataSource.png)
 
+## Custom endpoint
+Workbooks supports getting data from any external source. If your data lives outside Azure you can bring it to Workbooks by using this data source type.
+
+To make a query control use this data source, use the _Query Type_ drop down to choose _Custom Endpoint_. Provide the appropriate parameters such as Http method, url, headers, url parameters and/or body.
```

Comment:
there should probably be a `note` section here or something that indicates anything set in the headers/parameters/etc will be visible to any user of the workbook.  Until we have support for secrets, anything that requires api-keys or anything else will have any of those keys/etc visible in the workbook.